src/Microsoft.DotNet.Cli.Utils/project.json

```diff
@@ -9,19 +9,25 @@
       "target": "project"
     },
     "Microsoft.DotNet.PlatformAbstractions": "1.0.1-beta-000933",
+
     "NuGet.Versioning": "3.6.0-rc-1984",
     "NuGet.Packaging": "3.6.0-rc-1984",
     "NuGet.Frameworks": "3.6.0-rc-1984",
-    "NuGet.ProjectModel": "3.6.0-rc-1984"
+    "NuGet.ProjectModel": "3.6.0-rc-1984",
+
+    "Microsoft.Build": "15.1.316-preview5",
+    "Microsoft.Build.Utilities.Core": "15.1.316-preview5"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.5": {
       "imports": [
         "portable-net45+wp80+win8+wpa81+dnxcore50"
       ],
       "dependencies": {
         "System.Diagnostics.Process": "4.1.0"
       }
+    },
+    "net46": {
```

Comment:
The problem here is that binding-redirects use this indirectly through Test.Utilities and we want to keep the binding-redirect tests. It was easier to add this back now than to tease things apart.